samples/snippets/revoke_dataset_access_test.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .. import update_dataset_access
+from .. import revoke_dataset_access
```

Comment:
This should be `from . import revoke_dataset_access` since it's the same directory as the test. (Also will need `__init__.py` for that to work.) If you go with the "copy" alternative (without `__init__.py`), then this should be `import revoke_dataset_access`.